components/coding/SkillRatingsComponent.tsx

```diff
@@ -78,19 +78,8 @@ export default function SkillRatingsComponent(props) {
   };
 
   return (
-    <div className="flex flex-col w-full overflow-auto-bg-scroll">
-      <div className="flex justify-center p-4">
-        <Button
-          label="Save"
-          onClick={() =>
-            saveSkillRatings({
-              variables: {
-                objects: transformSkillRatingForDB(skillRatings, user),
-              },
-            })
-          }
-        />
-      </div>
+    <div className="flex bg-slate-900 flex-col w-full overflow-auto-bg-scroll">
+      <div className="flex justify-center p-4"></div>
```

Comment:
empty div? why not put those styles in the parent div?

Reply:
will do! thank you for catching this.

Reply:
I've been pairing with Kari on Tailwind styling to avoid silly div patterns like this. Still love tailwind but it can get messy quickly if im not careful lol